setup.py

```diff
@@ -24,24 +24,24 @@
       install_requires=[
           "requests>=2.31.0,<3",
           "pyjwt>=2,<3",
-          "arrow>=0.15.4,<0.16",
+          "arrow>=0.15.4,<2",
```

Comment:
Did the interface not change at all in 1.0?

Reply:
It changed, just not in a way that impacts us:
https://github.com/arrow-py/arrow/issues/832